app/assets/stylesheets/components/page_specific/_home_top.scss

```diff
@@ -0,0 +1,193 @@
+// Banner at the top of the home page
+//
+// Styleguide Home Top
+
+.home-top {
+  background: image_url('illustrated_people.png') 0 100% no-repeat;
+  background-size: 135%;
+  margin: 0;
+  position: relative;
+  width: 100%;
+  display: block;
+  padding-bottom: 50%;
+
+  @include respond-to($mq-m) {
+    background-position: 165% 108%;
+    background-size: 70%;
+    padding-bottom: 0;
+  }
+
+  @include respond-to($mq-l) {
+    background-size: 65%;
+    background-position: 141% 100%;
+
+    @if ($responsive == false) {
+      width: $mq-l;
+      margin: 0 auto;
+      float: none;
+    }
+  }
+
+  @include respond-to($mq-xl) {
+    background: none;
+    overflow: hidden;
+  }
+}
+
+.home-top-trust {
```

Comment:
We're specifying two components here in one file, which is against our convention. Should `home-top-trust` be a child of `home-top`, or should it be moved into its own component?

Or should `home-top` styles be moved into the `home-top` layout?